bin/build_and_run_all_tests

```diff
@@ -103,10 +104,33 @@ then
 	CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_NETCDF_PARALLEL=OFF"
 fi
 
-
+if [[ "x${PDI_SYSTEM}" =~ ^xmacos ]]
+then
+	#Workaround: not all plugins work on macos.
+	CMAKE_FLAGS="${CMAKE_FLAGS} \
+-DBUILD_BENCHMARKING=OFF \
+-DBUILD_DECL_HDF5_PLUGIN=ON \
+-DBUILD_DECL_NETCDF_PLUGIN=OFF \
+-DBUILD_DEISA_PLUGIN=OFF \
+-DBUILD_DOCUMENTATION=OFF \
+-DBUILD_HDF5_PARALLEL=ON \
+-DBUILD_FORTRAN=OFF \
+-DBUILD_MPI_PLUGIN=ON \
+-DBUILD_NETCDF_PARALLEL=ON \
+-DBUILD_PYCALL_PLUGIN=OFF \
+-DBUILD_SERIALIZE_PLUGIN=ON \
+-DBUILD_SET_VALUE_PLUGIN=ON \
+-DBUILD_TESTING=ON \
+-DBUILD_TRACE_PLUGIN=ON \
+-DBUILD_USER_CODE_PLUGIN=ON \
+-DUSE_HDF5=SYSTEM \
+-DUSE_NetCDF=SYSTEM \
+-DCMAKE_CXX_FLAGS=\"-Wno-unqualified-std-cast-call -undefined dynamic_lookup\""
+#          -DCMAKE_CXX_STANDARD=${{matrix.cxx_std}} \
+fi
 
 # Configure, build & test
 
-cmake -DDIST_PROFILE=Devel ${CMAKE_FLAGS} "${SRCDIR}"
+cmake -DDIST_PROFILE=Devel "${CMAKE_FLAGS}" "${SRCDIR}"
```

Comment:
The `CMAKE_FLAGS` is a space separated list of parameters, not one single parameter. This shouldn't be quoted